node/tests/PubSub.test.ts

```diff
@@ -66,13 +68,63 @@ describe("PubSub", () => {
         cmdCluster = standaloneAddresses
             ? await ValkeyCluster.initFromExistingCluster(
                   parseEndpoints(standaloneAddresses),
+                  async (addresses: [string, number][]) => {
```

Comment:
same here - track where standalone and cluster clients are used

Reply:
Updated